mtglabels/generator.py

```diff
@@ -125,23 +142,15 @@
 
 class LabelGenerator:
 
-    DEFAULT_OUTPUT_DIR = Path(os.getcwd()) / "output"
-
     COLS = 4
     ROWS = 15
     MARGIN = 200  # in 1/10 mm
     START_X = MARGIN
     START_Y = MARGIN
 
-    PAPER_SIZES = {
-        "letter": {"width": 2790, "height": 2160, },  # in 1/10 mm
-        "a4": {"width": 2970, "height": 2100, },
-    }
-    DEFAULT_PAPER_SIZE = "letter"
-
     def __init__(self, paper_size=None, output_dir=None):
         self.paper_size = paper_size or DEFAULT_PAPER_SIZE
```

Comment:
```suggestion
        self.paper_size = paper_size or self.DEFAULT_PAPER_SIZE
```

Ahhh, I think this is the bug

Reply:
ok, but the same error is in the line

`self.output_dir = Path(output_dir or DEFAULT_OUTPUT_DIR)`

it just doesn't fail at runtime since `output_dir`  it is never passed `None`.

I think the simplest change may be (see PR)

Change
`def __init__(self, paper_size=None, output_dir=None)`

to
`def __init__(self, paper_size=DEFAULT_PAPER_SIZE, output_dir=DEFAULT_OUTPUT_DIR)`

Then one can simplify

```py
self.paper_size = paper_size or DEFAULT_PAPER_SIZE
self.output_dir = Path(output_dir or DEFAULT_OUTPUT_DIR)
```

to

```py
self.paper_size = paper_size
self.output_dir = Path(output_dir)
```